src/Core/Managed/Shared/Extensibility/Implementation/TelemetryProcessorChainBuilder.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    /// <summary>
+    /// Represents an object used to Build a TelemetryProcessorChain.
+    /// </summary>
+    public sealed class TelemetryProcessorChainBuilder
+    {
+        private List<Func<ITelemetryProcessor, ITelemetryProcessor>> factories;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TelemetryProcessorChainBuilder" /> class.
+        /// </summary>
+        public TelemetryProcessorChainBuilder()
```

Comment:
styling: you don't need explicit constructor just to instantiate one list, put initialization to the filed definition